### PR TITLE
Exec cms and poll create fixes

### PR DESCRIPTION
**Pull request by tyler17**
### Link to Clubhouse story

Executive improvements:
https://app.clubhouse.io/dux-makerdao/story/47/create-executive-ui-whitelisted-to-governance-facilitators-load-from-markdown-in-github
Polling improvements: https://app.clubhouse.io/dux-makerdao/story/48/create-poll-ui-refresh-whitelisted-to-governance-facilitators-load-from-markdown-in-github

### What does this PR do?

1. Ignore executive votes with invalid address and invalid dates.
2. Improves formatting in the polling/create page

### Steps for testing:
Polling changes:
1. Go to the /polling/create page.  input a url pointing to a poll, e.g. https://raw.githubusercontent.com/makerdao/community/master/governance/polls/MIP9%20Community%20Greenlight%20Poll%20-%200xBTC%20-%20July%205%2C%202021.md
2. Confirm that it still displays poll info correctly, but improves formatting of options and content markdown, and cleans up the create confirmation modal.

The executive changes are harder to test in the UI, but I've added automated tests to check that the new parseExecutive function ignores execs with invalid addresses and invalid dates.

### Screenshots (if relevant):
Screen Shot 2021-07-08 at 2 12 10 PM
Screen Shot 2021-07-08 at 2 12 18 PM

### CODE REVIEW

**Review by adamgoth — APPROVED** [review]
Strictly a cosmetic thing but the `<CreateText />` component that displays the parsed sections looks like it could use a little bit of padding. Looks good to me otherwise!

File: lib/api.ts [hunk 1]
```diff
@@ -15,13 +15,59 @@ import { parsePollMetadata } from './polling/parser';
 import { fetchGitHubPage } from './github';
 import fs from 'fs';
 import { config } from './config';
+import { ethers } from 'ethers';
+
+export function parseExecutive(
```
Comment (rafinskipg) [inline lib/api.ts:20]: lib/api is starting to grow, 

A suggested refactor could be 

```
lib/api
   /executives.ts
      [here include the code for parsing executives and stuff ]

```

Comment (rafinskipg) [reply lib/api.ts:20]: note "api" is a folder

**Review by rafinskipg — APPROVED** [review]

**Comment by rafinskipg** [comment]
Notes: 
- When a user inputs a URL and clicks validate there is no visual feedback , could be good if : The button said "Loading", if it fails we show an error.

**Comment by rafinskipg** [comment]
Added the padding and the loading UI fix 🙏

**Pull request merged**